test/Benchmarks/src/Numeric.enso

```diff
@@ -22,24 +24,34 @@ bench =
     integers = Vector.new vector_size _-> faker.integer -1000000000 1000000000
 
     funs.map pair->
-        name = pair.at 0
+        name = "decimal " + pair.at 0
         fun = pair.at 1
-        IO.println <| "Benchmarking decimal " + name
-        Bench.measure (decimals.map fun) name iter_size num_iterations
+        IO.println <| "Benchmarking " + name
```

Comment:
I think we need to get rid of such `println`s, because they will start being printed also when only gathering benchmark structure.
```suggestion
```

Reply:
Yes. I think that as well. They are, indeed, printed during the benchmark gathering.

Reply:
Another thing is that initialization which is relatively heavy also happens at the gathering step, slowing it down a bit.

But I don't think we have a good solution for this yet. I have an idea how we might do this, but I don't think it's in the scope of this PR.

Reply:
It's not in scope of this PR, but it might be in the scope of https://github.com/enso-org/enso/pull/7519

Reply:
The *setup* and *teardown* approaches related discussion is at https://github.com/enso-org/enso/pull/7519#pullrequestreview-1573226446